__tests__/app.test.js

```diff
@@ -69,16 +69,90 @@ describe("GET /api/reviews", () => {
         });
       });
   });
-  test("200: reviews are sorted by newest first", () => {
+  test("200: reviews are sorted by newest first by default", () => {
     return request(app)
       .get("/api/reviews")
+      .expect(200)
       .then(({ body: { reviews } }) => {
-        expect(reviews).toBeSortedBy(reviews.created_at, {
+        expect(reviews).toBeSortedBy("created_at", {
           descending: true,
           coerce: true,
         });
       });
   });
+  describe("Queries", () => {
+    describe("Category", () => {
+      test("200: Filter by specified category", () => {
+        return request(app)
+          .get("/api/reviews?category=social deduction")
+          .expect(200)
+          .then(({ body: { reviews } }) => {
+            expect(reviews).toHaveLength(11);
+            reviews.forEach((review) => {
+              expect(review.category).toBe("social deduction");
+            });
+          });
+      });
+      test("200: filtering by a category that has no reviews returns an empty array", () => {
+        return request(app)
+          .get("/api/reviews?category=drinking")
+          .expect(200)
+          .then(({ body: { reviews } }) => {
+            expect(reviews).toEqual([]);
+          });
+      });
+      test("200: does not allow SQL injection", () => {
```

Comment:
I dont know if this is what I would call this test, is it a non-existent category name, in which case should this be a 404?